environment/provider.tf

```diff
@@ -70,7 +67,6 @@ provider "aws" {
     role_arn     = "arn:aws:iam::515688267891:role/${var.DEFAULT_ROLE}"
     session_name = "terraform-session"
   }
-  version = "2.70.0"
```

Comment:
Were we locking this to 2.7 for a specific reason @jamesrwarren? I seem to have something at the back of my head telling me so but no idea why.

Reply:
Ignore me - have just seen it in the versions.tf!

Reply:
3.0 Introduced some changes that need some rework, I'll put a ticket into the backlog to do the upgrade at some point next week